content/news/release-20150424.md

```diff
@@ -0,0 +1,64 @@
+We are very pleased to announce the Spring release of OpenRA!
+
+This release a huge step forward for the Dune 2000 mod in particular. It includes just shy of 1000 commits by 53 contributors, the culmination of six months of relentless work. With more than 2000 files and 210,000 changed lines it is one of our busiest development cycles to date.
+
+A large development focus has been on the Dune 2000 mod, which contains the most visible changes. A new sidebar, sandworms and Carryalls are the big new features, which should make any Dune 2000 player feel more at home in our mod. Additionally, the OpenRA.Utility now supports importing maps from the original Dune 2000 game, so we hope that the number of maps and of course the number of players will increase in the near future!
+
+The other mods were not neglected, though. In Red Alert, the standout feature is the support of playable factions such as Russia, England, France, Ukraine and Germany. In both Red Alert and Tiberian Dawn, new single-player missions are available that will keep you occupied.
+
+Of course, we have not only improved the mods themselves. Here is a list of changes that affect the game in general:
+
+* The server browser shows and allows joining games from any installed mod.
+* It is now possible to disable the "short game" mode that was forced in the past.
+* The player name and colour can now be set up in the settings dialog.
+* When creating a server, the game informs the host about port forwarding issues.
+* Left-click mouse orders have been markedly improved in this release.
+* Some missions now support difficulty levels.
+* A huge number of smaller gameplay fixes and improvements have been added. A few examples:
```

Comment:
I would personally tighten this to "A huge number of smaller gameplay fixes and improvements, including:".  Not a big deal if you can't be bothered changing it, though.